client/src/test/kotlin/com/spectralogic/tpfr/client/ReadFileFromResources.kt

```diff
@@ -20,7 +20,7 @@ import org.slf4j.LoggerFactory
 import java.io.IOException
 
 internal object ReadFileFromResources {
-    private val LOG = LoggerFactory.getLogger(ClientImpl::class.java)
+    private val LOG = LoggerFactory.getLogger(TpfrClientImpl::class.java)
```

Comment:
Shouldn't this logger be ReadFileFromResources?

Reply:
good catch